module-4/backend/week_one.md

```diff
@@ -5,16 +5,31 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+It's a tie between using 'var', 'return', and array prototype methods. I haven't yet used constructor functions, but I imagine they're really important for OOP.
 2. What are some tools to help debug JavaScript code?
+console.log()
+debugger
+pryjs
 3. What are some tools you need in order to unit test your JavaScript?
+mocha & chai
 4. What is the syntax for invoking a function?
+nameOfFunction()
 5. What's the difference between `==` and `===` in JavaScript?
+`==` will eval to true for different datatypes (eg. "1" == 1), `===` is exactly equal to, meaning it will only evaluate to true if both values are equal and the same datatype.
 6. What's the difference between asynchronous and synchronous JavaScript? 
+Asynchronous JS continues to run, no matter order and regardless of req/response (eg. in our tests we need to provide `done()` to indicate that the test can move on); synchronous is much like Ruby where it runs in the order (and/or hierarchy) that we indicate
```

Comment:
Great! Just remember that it does run in order, it just doesn't wait for async functions to resolve/evaluate before moving on to the next line 👍